CONTRIBUTING.md

```diff
@@ -173,6 +173,47 @@ The `legacy_foo` arg was removed
 :::
 ```
 
+## Style and idioms
+
+For the most part, we just accept whatever the code formatters do, so there
+isn't much style to enforce.
+
+Some miscellanous style, idioms, and conventions we have are:
+
+### Markdown/Sphinx Style
+
+* Use colons for prose sections of text, e.g. `:::{note}`, not backticks.
+* Use backticks for code blocks.
+* Max line length: 100.
```

Comment:
Having `.editorconfig` in the root of the repo to ensure that this is easy to follow would be great. `nvim` for example has native support for it, so do many other editors.

Reply:
Oh neat. I added a basic editorconfig

I don't see a setting for line length, though?

Reply:
It has `max_line_length`, no? https://devhints.io/editorconfig.

But I've seen some discussions about people wanting to drop it, but I think we are using it.